### DISCUSSION TITLE
Need Changelog Url

### DISCUSSION THREAD

**Discussion by krishnaTORQUE** [twbs/bootstrap#31890] [Q&A] [answered]
Where is the Changelog? I am unable to find changelog.
And popperjs does not require anymore for `4.5.3`?

**Comment by XhmikosR** [comment, marked as answer]
https://github.com/twbs/bootstrap/releases/tag/v4.5.3

Popper.js is still required if you use our JavaScript file. But we also offer a bundle file with Popper.js included, and we switched to suggesting the bundle file, that is all.

**Reply by krishnaTORQUE** [reply]
you mean `/node_modules/bootstrap/dist/js/bootstrap.bundle.min.js` path to use in nodejs ?
is this the path include popperjs?
if yes does it also include jquery ?

**Comment by XhmikosR** [comment]
https://getbootstrap.com/docs/4.5/getting-started/contents/#js-files

**Reply by krishnaTORQUE** [reply]
thank you so much
